#
# This file is autogenerated by pip-compile with Python 3.12
# by the following command:
#
#    pip-compile ./requirements.in
#
blinker==1.8.2
    # via flask
cachetools==5.5.0
    # via google-auth
certifi==2024.8.30
    # via requests
charset-normalizer==3.4.0
    # via requests
click==8.1.7
    # via flask
flask==3.0.0
    # via -r ./requirements.in
google-api-core[grpc]==2.22.0
    # via
    #   google-cloud-core
    #   google-cloud-datastore
    #   google-cloud-ndb
google-auth==2.17.3
    # via
    #   google-api-core
    #   google-cloud-core
google-cloud-core==2.4.1
    # via google-cloud-datastore
google-cloud-datastore==2.15.1
    # via google-cloud-ndb
google-cloud-ndb==2.1.1
    # via -r ./requirements.in
googleapis-common-protos==1.65.0
    # via
    #   google-api-core
    #   grpcio-status
grpcio==1.67.1
    # via
    #   google-api-core
    #   grpcio-status
grpcio-status==1.62.3
    # via google-api-core
idna==3.10
    # via requests
itsdangerous==2.2.0
    # via flask
jinja2==3.1.4
    # via flask
markupsafe==3.0.2
    # via
    #   jinja2
    #   werkzeug
proto-plus==1.25.0
    # via
    #   google-api-core
    #   google-cloud-datastore
protobuf==4.25.5
    # via
    #   google-api-core
    #   google-cloud-datastore
    #   google-cloud-ndb
    #   googleapis-common-protos
    #   grpcio-status
    #   proto-plus
pyasn1==0.6.1
    # via
    #   pyasn1-modules
    #   rsa
pyasn1-modules==0.4.1
    # via google-auth
pymemcache==4.0.0
    # via google-cloud-ndb
pytz==2024.2
    # via google-cloud-ndb
redis==4.6.0
    # via google-cloud-ndb
requests==2.28.2
    # via google-api-core
rsa==4.9
    # via google-auth
six==1.16.0
    # via google-auth
urllib3==1.26.20
    # via requests
werkzeug==3.0.6
    # via flask

# The following packages are considered to be unsafe in a requirements file:
setuptools
